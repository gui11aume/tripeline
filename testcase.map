CTGTCCTGCACACGCGAAAA	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3R:-:24329772:91
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	3R:-:7485179:78
NGGTCTGTCCAAACTCTAGG	GCTATAGCNTCTAGTCCGGGACNNTTGTCTGGCGTATCCAGACNCNTGCCTAGAGTTTGGACAGACCCAGATCGNAANAGCGTCGTGTAGG		0:0:0:0:0:0:0:0:0:0:0	-
TGATCTGGGCTCTTATCGGT	GGTACAATTATTTGAATTTTTGAGGTGTACATTACGTGGGGTGAGTTTGGGGATTGGACATAGATTTAGTATTAATATTAAGAGGTAAGTA		!	-
CAACGCACACAGTCGATTTA	TAGGGATCCTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTACCCAGCTTTTGTTCCCGTAACTATAACGGTCCTAAGGTAGCGAACTT		1	pT2:+:522:91
TCGCATTCAAGCGTCTCTGG	TCAAGTCTGTGAGAATCCAAAATGTCCGGATAGATTTGAAGAACCTGTTCTCAAACAAAAGGAGAAGGAATATATGAAAGATTTGTCCTGT		1	2L:-:16332345:91
CGAAAAGCATAAACTGGGTG	TAGGGATCCTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTACCCAGCTTTTGTTCCCGTAACTATAACGGTCCTAAGGTAGCAACTTT		0:0:0:1	pT2:+:522:85G1AC2
CTGGAAGCCGACAGTCTTAA	TATCCAAACCAAGTTTCAATTTTACAGATAACAAGTTCGATCTAAGTTTTATGAGAACTTTCGATACCTGAGACCAATGATCTGGATTTAG		!	-
AAGTCTGGATCAACTTTATT	TTTTAAATATGTAATACTCGGTACTCAGTGAACACCACGGGAAGGTCATTACAATTGTATTTAAAGGAGAAAAAAACAGTTGGCGCACTTT		!	-
TTGACTAGCGATACTAGTGG	TGAAGCGAAGTTCCTGCTTTCTGGTCTCCGTCAGCAGTTGATCGTAATCCCTGCG		1	3L:-:846835:55
TAGGGTAGTTTCCAATTAGA	TATCTACCGTAGTTTCATCCCCATCATCCTCATCGTCATCGTCATCATCCGCATCCTCGTCGGCATCCCCTTCGCCACTGCTCCGCCTTTG		0:0:0:0	-
ACATAAGGTACTGGACTCGT	CCCGAACCCGAAACCGACATTATCATTACTGCATATATAGTGTGTAATGGC		1	3R:+:17422872:51
NGCTGCGTGTACAACGTCTA	TAGGGATCNTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTNCNCAGCTTTTGTTCCCGTAACTATAACGGTNCTAAGGTAGCGAACTT		0:0:0:0:1:0:0:0	pT2:+:522:8C34A1C28C16
ATTAGGCCACCAGGACTCAA	TAGGGATCCTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTACCCAGCTTTTGTTCCCGTAACTATAACGGTCCTAAGGTAGCGAACTT		1	pT2:+:522:91
CTGTCCTGCACACGCGAAAA	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3R:-:24329772:91
GTCTTTTAGTCGATGCAGT	AGCAATGCTATATTTTGGGAGCTTTTATGTGATTATCACATCACATTTACCAAAGACTAAGACAGACAGCCATTTTCTATTGTATATATAT		0:0:0:1	2R:-:7317792:61CA5A22
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	3R:-:7485179:78
NGGTCTGTCCAAACTCTAGG	GCTATAGCNTCTAGTCCGGGACNNTTGTCTGGCGTATCCAGACNCNTGCCTAGAGTTTGGACAGACCCAGATCGNAANAGCGTCGTGTAGG		0:0:0:0:0:0:0:0:0:0:0	-
TGATCTGGGCTCTTATCGGT	GGTACAATTATTTGAATTTTTGAGGTGTACATTACGTGGGGTGAGTTTGGGGATTGGACATAGATTTAGTATTAATATTAAGAGGTAAGTA		!	-
CAACGCACACAGTCGATTTA	TAGGGATCCTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTACCCAGCTTTTGTTCCCGTAACTATAACGGTCCTAAGGTAGCGAACTT		1	pT2:+:522:91
TCGCATTCAAGCGTCTCTGG	TCAAGTCTGTGAGAATCCAAAATGTCCGGATAGATTTGAAGAACCTGTTCTCAAACAAAAGGAGAAGGAATATATGAAAGATTTGTCCTGT		1	2L:-:16332345:91
CGAAAAGCATAAACTGGGTG	TAGGGATCCTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTACCCAGCTTTTGTTCCCGTAACTATAACGGTCCTAAGGTAGCAACTTT		0:0:0:1	pT2:+:522:85G1AC2
CTGGAAGCCGACAGTCTTAA	TATCCAAACCAAGTTTCAATTTTACAGATAACAAGTTCGATCTAAGTTTTATGAGAACTTTCGATACCTGAGACCAATGATCTGGATTTAG		!	-
AAGTCTGGATCAACTTTATT	TTTTAAATATGTAATACTCGGTACTCAGTGAACACCACGGGAAGGTCATTACAATTGTATTTAAAGGAGAAAAAAACAGTTGGCGCACTTT		!	-
TTGACTAGCGATACTAGTGG	TGAAGCGAAGTTCCTGCTTTCTGGTCTCCGTCAGCAGTTGATCGTAATCCCTGCG		1	3L:-:846835:55
TAGGGTAGTTTCCAATTAGA	TATCTACCGTAGTTTCATCCCCATCATCCTCATCGTCATCGTCATCATCCGCATCCTCGTCGGCATCCCCTTCGCCACTGCTCCGCCTTTG		0:0:0:0	-
ACATAAGGTACTGGACTCGT	CCCGAACCCGAAACCGACATTATCATTACTGCATATATAGTGTGTAATGGC		1	3R:+:17422872:51
NGCTGCGTGTACAACGTCTA	TAGGGATCNTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTNCNCAGCTTTTGTTCCCGTAACTATAACGGTNCTAAGGTAGCGAACTT		0:0:0:0:1:0:0:0	pT2:+:522:8C34A1C28C16
ATTAGGCCACCAGGACTCAA	TAGGGATCCTCTAGCTAGAGTCGACCTCGAGGGGGGGCCCGGTACCCAGCTTTTGTTCCCGTAACTATAACGGTCCTAAGGTAGCGAACTT		1	pT2:+:522:91
replace_me	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3R:-:24329772:91
CTGTCCTGCACACGCGAAAA	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3R:-:24329771:91
CTGTCCTGCACACGCGAAAA	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3L:-:24329772:91
CTGTCCTGCACACGCGAAAA	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3L:-:24329772:91
replace_me	CAAAGTGTCAAAGTGATGGCCGCCAGGAGAGTGTCACGGCTGACTATGGGTTGTTGCCACTGTCAACACGGCAATGACAGGGCACAAACCA		1	3L:-:24329772:91
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	3R:-:7485180:78
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	3R:-:7485180:78
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	3R:-:7485180:78
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		0:0:0:0	-
no_canonical	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	3R:-:7485180:78
GGGGGCTACAGTGGACGGGG	TACATATGTATATGGCCACCTGCCAAAAATCCAGATCTCTTCTAGAGATGCCACCAGTGTATTGTCTTGGTTCAATGA		1	nowhere:-:0:78
