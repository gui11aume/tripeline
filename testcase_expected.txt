TCGCATTCAAGCGTCTCTGG	2L	-	16332436	2
TTGACTAGCGATACTAGTGG	3L	-	846890	2
GGGGGCTACAGTGGACGGGG	3R	-	7485258	7
ACATAAGGTACTGGACTCGT	3R	+	17422872	2
ATTAGGCCACCAGGACTCAA	pT2	+	522	2
CAACGCACACAGTCGATTTA	pT2	+	522	2
CGAAAAGCATAAACTGGGTG	pT2	+	522	2
NGCTGCGTGTACAACGTCTA	pT2	+	522	2
